guides/common/modules/proc_creating-custom-provisioning-snippets.adoc

```diff
@@ -1,25 +1,24 @@
 [id="Creating_Custom_Provisioning_Snippets_{context}"]
 = Creating custom provisioning snippets
```

Comment:
The title strikes me as odd. Shouldn't we tell the user the use case, rather than the means to do it? I suggest a title such as "Running custom code while provisioning hosts". I know that the custom code is run before or after provisioning, but I think that this works for a title.

Then I would expect a lead-in sentence: "You can run custom code before or after provisioning the host by creating a custom provisioning snippet."

I can't recall the exact source but I was taught not to use 'and/or' because this construction can cause translation issues. It is also ambiguous. The user has to figure out whether you mean 'before and after', 'before or after', or 'before, after, or both'.